frontend/pages/SoftwarePage/SoftwareInventory/_styles.scss

```
@@ -0,0 +1,4 @@
+.software-inventory {
+  @include vertical-page-tab-panel-layout;
+  margin-top: $gap-page-component; // Required as these Tabs don't use TabPanel
```

Stylelint (17.9.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)